tests/integration/monitoring/test-monitoring.go

```diff
@@ -128,6 +128,9 @@ func testPodsAreReady() {
 func testTargetsAreHealthy() {
 	timeout := time.After(3 * time.Minute)
 	tick := time.NewTicker(30 * time.Second)
+	var labelsToBeIgnored = promAPI.Labels{
+		"namespace": "e2e-event-mesh", // e2e test pod
```

Comment:
I am wondering. If the test causes problems with the core-test-external-solution test, would it make sense to `not` run the monitoring test in `parallel` ? Maybe other tests are also creating dynamic targets and have the same problem.

Reply:
If that's the case, then running monitoring tests, `not` in parallel will not help as the targets created by the other tests get deleted once they are done. Ignoring those targets is a better option, IMO which doesn't impact the total execution time of the test suite.